templates/web/base/report/_item.html

```diff
@@ -43,9 +43,13 @@
   [% PROCESS 'report/_item_extra_class.html' %]
 [%~ CATCH file ~%]
 [%~ END ~%]
-
 <li class="item-list__item item-list--reports__item [% item_extra_class %]"
     data-report-id="[% problem.id | html %]" data-lastupdate="[% problem.lastupdate %]" id="report-[% problem.id | html %]">
+[% IF relevant_staff AND (c.user.has_permission_to('assign_report_to_user', problem.bodies_str_ids)) %]
+    [% UNLESS ((c.req.uri.path.match('\/my\/')) OR  (c.req.uri.path.match('\/around'))) %]
+        <input type='checkbox' class='bulk-assign' name='bulk-assign-reports' form='bulk-assign-form' value='[%- problem.id -%]' style='float:right; margin:2em'>
```

Comment:
These checkboxes appear to unselectable (at least in Chrome/Firefox), trying to do so loads the report detail page instead. The assignment stars appear to still work okay.